src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -6,12 +6,77 @@
   justify-content: stretch;
   position: relative;
 
-  .title {
-    font-weight: var(--medium);
-    font-size: var(--text-xl);
-    line-height: 30px;
-    color: var(--text-default);
-    text-align: left;
+  .titleWrapper {
+    display: flex;
```

Comment:
this css is not really changing this much, I just added this titleWrapper and display: flex, the rest is because I moved nested things but I didn't change the rest of the styles